pkg/mysqlsh/mysqlsh.go

```diff
@@ -0,0 +1,173 @@
+package mysqlsh
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"regexp"
+
+	"github.com/pkg/errors"
+	k8sexec "k8s.io/utils/exec"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+
+	"github.com/percona/percona-server-mysql-operator/pkg/innodbcluster"
+)
+
+type mysqlsh struct {
+	uri  string
+	exec k8sexec.Interface
+}
+
+var sensitiveRegexp = regexp.MustCompile(":.*@")
```

Comment:
I think this regexp can cause problems while printing error. Does mysqlsh really prints password in stdout/stderr?

Reply:
it's not the `mysql-shell` but us that print password to stdout/stderr (without this regexp). We pass instance URI to various commands in `mysqlsh` and we print the command and its arguments if there's an error. This URI has password in most cases.